abci/example/kvstore/tool/make_val_set_change_tx.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/line/ostracon/abci/example/kvstore"
+	"github.com/line/ostracon/config"
+	"github.com/line/ostracon/crypto/encoding"
+)
+
+func main() {
+	c := config.DefaultConfig()
+	c.SetRoot(os.Getenv("HOME") + "/" + config.DefaultOstraconDir)
```

Comment:
Thanks, I've re-considered and checked the others.
The below helps me.
* https://github.com/line/ostracon/blob/main/cmd/ostracon/main.go#L47
1. `os.ExpandEnv`
2. `filepath.Join`

How do you like to use it?
* [ ] `os.UserHomeDir`
* [ ] `os.ExpandEnv`
* [ ] `path.Join`
* [ ] `filepath.Join`

Reply:
Basically yes, I think we can also change this. It's in a state that will cause a conflict if it's modified in Tendermint because this line has already been [rewritten by us](https://github.com/line/ostracon/commit/f10d45a9381ef72ddbffb15c66e8c8bd7231f253). And, it seems not to be a conflict that would confuse in the merge operation.